multicluster/controllers/multicluster/resourceexport_controller.go

```diff
@@ -260,28 +272,29 @@ func (r *ResourceExportReconciler) refreshServiceResourceImport(
 		}
 		return newResImport, true, nil
 	}
-	// TODO: check ClusterIPs difference if it is being used in ResrouceImport later
+	// TODO: check ClusterIPs difference if it is being used in ResourceImport later
 	convertedPorts := svcPortsConverter(resExport.Spec.Service.ServiceSpec.Ports)
 	if !apiequality.Semantic.DeepEqual(newResImport.Spec.ServiceImport.Spec.Ports, convertedPorts) {
 		undeletedItems, err := r.getUndeletedResourceExportItems(resExport)
 		if err != nil {
 			klog.ErrorS(err, "fail to list ResourceExports, retry later")
 			return newResImport, false, err
 		}
-		// when there is only one Service ResourceExport, ResourceImport should reflect the change
-		// otherwise, it should skip it due to it's impossible to determine which one is desired.
+		// When there is only one Service ResourceExport, ResourceImport should reflect the change
+		// otherwise, it should always return error so controller can retry later if admin fix the issue
```

Comment:
How about "assuming users can fix the conflicts"?

Reply:
done